exercises/practice/mazy-mice/src/test/java/MazeGeneratorTest.java

```diff
@@ -85,6 +90,7 @@ public void theMazeHasSingleExitOnTheRightSideOfTheMaze() {
 
     @Disabled("Remove to run test")
     @Test
+    @DisplayName("Consecutive maze generations produce different mazes")
```

Comment:
```suggestion
    @DisplayName("Maze is different each time it is generated")
```

Let's try to keep it consistent with the others